code/game/objects/structures/aliens.dm

```diff
@@ -1,428 +1,428 @@
-/* Alien shit!

```

Comment:
Uhhhh. What did you do?

Reply:
I explained above that this file decided that it needed to be in my PR despite no changes.